new term: Schedule (#1254)
temporalio/documentation: docs/tctl/schedule/trigger.md

---
id: trigger
title: tctl schedule trigger
sidebar_label: trigger
description: How to trigger a Schedule Action using tctl
tags:
  - tctl
---

> It's a common desire to want a Schedule to start one Workflow run immediately, regardless of its configured Spec.

Reviewer: ```suggestion
It is a common use case to want a Schedule to start one Workflow run immediately, regardless of its configured Spec.
```

Reviewer: Or: "Starting a Workflow Run immediately with a Schedule, regardless of its configured Spec, is a common use case."

```shell
$ tctl schedule trigger --sid 'my-schedule-id'
```

> Note that the action that it takes is subject to the Overlap Policy of the Schedule by default: if the overlap policy is `Skip` and there already is a Workflow running, the immediately-triggered run will be skipped! Likewise, if the overlap policy is `BufferAll`, the triggered run will be buffered behind one or more runs.

Reviewer: ```suggestion
Note that the action that it takes is subject to the Overlap Policy of the Schedule by default: if the overlap policy is `Skip` and there already is a Workflow running, the triggered Action to start the next Workflow Run will be skipped! 
Likewise, if the overlap policy is `BufferAll`, the triggered run will be buffered behind one or more runs.
```

---
id: trigger
title: tctl schedule trigger
sidebar_label: trigger
description: How to trigger a Schedule Action using tctl
tags:
  - tctl
---

Starting a Workflow Run immediately with a Schedule, regardless of its configured Spec, is a common use case.

```shell
$ tctl schedule trigger --sid 'my-schedule-id'
```

> Note that the action that it takes is subject to the Overlap Policy of the Schedule by default: if the overlap policy is `Skip` and there already is a Workflow running, the triggered Action to start the next Workflow Run will be skipped!
> Likewise, if the overlap policy is `BufferAll`, the triggered run will be buffered behind one or more runs.
> If you really want it to run right now, you can override the overlap policy for this request:

Reviewer 2: ```suggestion
Note that the action that it takes is subject to the Overlap Policy of the Schedule by default: if the overlap policy is `Skip` and a Workflow is already running, the triggered Action to start the next Workflow Run is skipped!
Likewise, if the overlap policy is `BufferAll`, the triggered run is buffered behind one or more runs.

If you really want it to run right now, you can override the overlap policy for this request:
```
